components/Account.php

```diff
@@ -563,4 +582,17 @@ protected function redirectForceSecure()
 
         return Redirect::secure(Request::path());
     }
+
+    /**
+     * Returns true if user is throttled.
+     * @return bool
+     */
+    protected function isRegisterThrottled()
+    {
+        if (!UserSettings::get('use_register_throttle', true)) {
```

Comment:
@daftspunk Should this setting default to `false` by default? Otherwise, we're going to be adding throttling on everyone who upgrades. If that's the case, we should probably make mention that it's enabled by default in the upgrade notes.